apps/remix-ide/src/app/plugins/parser/code-parser.tsx

```diff
@@ -167,6 +167,24 @@ export class CodeParser extends Plugin {
       await this.handleChangeEvents()
     })
 
+    this.on('solidity', 'compilerQueryParamsUpdated', async (file) => {
```

Comment:
This event doesn't have `file` as parameter. (I think same for the events below.